code/modules/events/patient_zero.dm

```diff
@@ -0,0 +1,55 @@
+/datum/round_event_control/patient_zero
+	name = "Patient Zero"
+	typepath = /datum/round_event/ghost_role/patient_zero
+	weight = 5
+
+	min_players = 10
+
+/datum/round_event/ghost_role/patient_zero
+	announceWhen	= 400
+
+	minimum_required = 1
+	role_name = "patient zero"
+
+	// 50% chance of being incremented by one
+	var/spawncount = 1
+	fakeable = TRUE
+
+/datum/round_event/ghost_role/patient_zero/announce(fake)
+	priority_announce("Unidentified lifesigns detected coming aboard [station_name()]. Secure any exterior access, including ducting and ventilation.", "Lifesign Alert", 'sound/ai/aliens.ogg')
```

Comment:
Probably should be unique considering it spawns an infectious zombie and is basically round ending

Reply:
I'll leave this to debate.